src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/services/StudyServiceImpl.java

```diff
@@ -47,11 +46,9 @@ public Study findOneStudyByPredicate(Predicate predicate) {
     }
 
     @Override
-    public Study findOneStudyByAccession(AccessionVersionEntityId id) {
+    public Study findOneStudyById(Long id) {
```

Comment:
You can use `CrudRepository::findOne(ID id)` directly here now.

Reply:
This method is used to bypass Spring AOP's methods that returns  released studies or non-deprecated only .